src/test/java/ibm/jceplus/junit/base/BaseTestMessageDigestClone.java

```diff
@@ -57,7 +57,7 @@ public void testUpdateCloneDifferentUpdate() throws Exception {
         byte[] digest1 = md.digest(input_2);
         byte[] digest2 = mdCopy.digest(input_3);
 
-        assertFalse("Digest of original matches clone's digest when it shouldn't", Arrays.equals(digest1, digest2));
+        assertFalse(Arrays.equals(digest1, digest2), "Digest of original matches clone's digest when it shouldn't");
```

Comment:
Should we use an `assertArraysEqual()` here and in all other instances in this test?